site/en/user_guide.md

```diff
@@ -457,12 +449,23 @@ docker run -u $(id -u) -v $PWD:/data -w /data -- ocrd/all:maximum ocrd-tesserocr
               (1)          (2)         (3)          (4)                            (5)
 ```
 
+<!--
+```sh
+docker run -u $(id -u) -v $PWD:/data -v $PWD/models:/usr/local/share/tessdata -w /data -- ocrd/all:maximum ocrd-tesserocr-segment-region -I OCR-D-IMG -O OCR-D-SEG-BLOCK
+           \_________/ \___________/ \______________________________________/ \______/ \_________________/ \___________________________________________________________/
+              (1)          (2)         						(3)          		 (4)          (5)									(6)
```

Comment:
What's (6)? Or why comment this out and not remove it?

Reply:
There are six elements in the list when using the examples in the comment (I was not sure if I maybe need that later) because it's using tesserocr and therefore mounting `usr/local/share/tessdata` additionally